staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -89,6 +89,12 @@ type KubeSchedulerConfiguration struct {
 	// with the extender. These extenders are shared by all scheduler profiles.
 	// +listType=set
 	Extenders []Extender `json:"extenders,omitempty"`
+
+	// DelayCacheUntilActive specifies when to start caching. If this value is true, it makes non-leader
+	// candidates not start with watching cache. Doing so will have slower failover with the benefit of
+	// lower memory overhead.
+	// Defaults to false.
+	DelayCacheUntilActive *bool `json:"delayCacheUntilActive,omitempty"`
```

Comment:
since the default is now false, can make this a non-pointer

Reply:
API bit lgtm otherwise

Reply:
Done